sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,1299 @@
+                                readonly example: "3/4/2021";
+                            };
+                            readonly Description: {
+                                readonly type: "string";
+                                readonly description: "The text description for the invoice line item";
+                                readonly example: "Consulting service";
+                            };
+                            readonly Quantity: {
+                                readonly type: "number";
+                                readonly description: "The quantity for this invoice line item";
+                                readonly example: "2";
+                            };
+                            readonly ProductCode: {
+                                readonly type: "string";
+                                readonly description: "Product code, product number, or SKU associated with the specific line item";
+                                readonly example: "A123";
+                            };
+                            readonly Tax: {
+                                readonly type: "number";
+                                readonly description: "Tax associated with each line item. Possible values include tax amount, tax %, and tax Y/N";
+                                readonly example: "10%";
+                            };
+                            readonly Unit: {
+                                readonly type: "string";
+                                readonly description: "The unit of the line item, e.g, kg, lb etc.";
+                                readonly example: "hours";
+                            };
+                            readonly UnitPrice: {
+                                readonly type: "number";
+                                readonly description: "The net or gross price (depending on the gross invoice setting of the invoice) of one unit of this item";
+                                readonly example: "$30.00";
+                            };
+                        };
+                    };
+                };
+            };
+        };
+    };
+};
+
+// @public
+export interface LayoutResult {
+    // (undocumented)
+    pages: DocumentPage[];
+    styles: DocumentStyle[];
+    tables: DocumentTable[];
+}
+
+// @public
+export type LengthUnit = string;
+
+// @public
+export interface ListModelsOptions extends OperationOptions {
+}
+
+// @public
+export interface ListOperationsOptions extends OperationOptions {
+}
+
+// @public
+export type ModelInfo = ModelSummary & {
+    docTypes?: {
+        [propertyName: string]: DocTypeInfo;
+    };
+};
+
+// @public
+export interface ModelSchema {
+    docTypes: {
+        [type: string]: {
+            fieldSchema: {
+                [k: string]: FieldSchema;
+            };
+        };
+    };
+    modelId: string;
+}
+
+// @public
+export interface ModelSummary {
+    createdDateTime: Date;
```

Comment:
I think the convention we use is `createdOn`

Reply:
This is exported from generated code. We might consider renames for these in future betas.

Reply:
That is totally fine by me. Let's create an issue to track all the renames to be done in the next beta.